front-end/src/ui-components/DismissableNotification.tsx

```diff
@@ -20,15 +20,15 @@ const DismissableNotification = ({ className, notification, onDismiss }: Props)
 	switch (status){
 	case ERROR:
 		color = 'red';
-		icon = 'ambulance';
```

Comment:
You didn't like my :ambulance: ?
Joking, I don't mind being less funky here.

Reply:
Haha, I went for the standard here